Build/Common.Build.Default.props

```diff
@@ -43,7 +43,6 @@
   <!-- Change configuration name to output to a different folder -->
   <!-- POGO instrumentation -->
   <PropertyGroup>
-    <IsPogoBuild />
```

Comment:
Should this actually be removed?  This seems like it probably default initializes IsPogoBuild to something.  I don't know msbuild well enough but I'm wary of this change if it was a drive by cleanup.